tools/stronghold/tests/api/test_compatibility.py

```diff
@@ -0,0 +1,318 @@
+import inspect
+import pathlib
+import tempfile
+import textwrap
+from typing import Any
+
+import api.compatibility
+
+
+def test_deleted_function(tmp_path: pathlib.Path) -> None:
```

Comment:
Q: With our current method, renaming the function (while keeping the args the same), will essentially be recognized as a deleted function? I think this is perfectly fine given we use function names as the unique key in the AST, just wanted to make sure. Perhaps we can encode this in a test as well?

Reply:
Yes, you are correct. Good point. I have added a test that verifies that a rose (or a function) by any other name, would look like a deleted rose (or function).